### ISSUE TITLE
Deploy fails inscrutably: "provided viceroy_version () is not a valid SemVer string"

### ISSUE THREAD

**Issue by matanlurey** [fastly/compute-actions#50]
```txt
Run fastly/compute-actions@v6
  with:
    project_directory: examples/hello-fastly
    cli_version: latest
    service_id: default
    verbose: false
  env:
    FASTLY_API_TOKEN: ***
Downloading 'fastly' (v10.8.10) from https://github.com/fastly/cli/releases/download/v10.8.10/fastly_v10.8.10_linux-amd64.tar.gz
/usr/bin/tar xz --warning=no-unknown-keyword -C /home/runner/work/_temp/e631099b-cccb-40e4-8f71-5e0[2](https://github.com/matanlurey/wasm-compute/actions/runs/8746083970/job/24002275917#step:2:2)1bd09216 -f /home/runner/work/_temp/fe6aab[3](https://github.com/matanlurey/wasm-compute/actions/runs/8746083970/job/24002275917#step:2:3)6-3e14-484a-b418-36c90252e7c7
/opt/hostedtoolcache/fastly/10.8.10/x6[4](https://github.com/matanlurey/wasm-compute/actions/runs/8746083970/job/24002275917#step:2:4)/fastly version
Error: The provided viceroy_version () is not a valid SemVer string.
Fastly CLI version v10.8.10 (7f3911[6](https://github.com/matanlurey/wasm-compute/actions/runs/8746083970/job/24002275917#step:2:6)4)
Built with go version go1.22.2 linux/amd64 (2024-04-1[8](https://github.com/matanlurey/wasm-compute/actions/runs/8746083970/job/24002275917#step:2:8))
```

<https://github.com/matanlurey/wasm-compute/actions/runs/8746083970/job/24002275917>

My configuration is pretty basic, it looks like this:
```yaml
name: deploy
on:
  push:
    branches:
      - main

jobs:
  deploy:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v4

    - name: Install Rust toolchain
      uses: dtolnay/rust-toolchain@stable
      with:
          targets: wasm32-wasi # WebAssembly target

    - name: Deploy examples/hello-fastly
      uses: fastly/compute-actions@v6
      with:
        project_directory: examples/hello-fastly
      env:
        FASTLY_API_TOKEN: ${{ secrets.FASTLY_API_TOKEN }}
```

**Comment by matanlurey** [comment]
Ah looks like this was fixed in https://github.com/fastly/compute-actions/pull/48, maybe there just needs to be a publish?

**Comment by kailan** [comment]
Hey @matanlurey – thanks for the report. This was indeed fixed in #48 and that was published as v7.

Could you update your workflow to reference the new release, `fastly/compute-actions@v7`, and confirm that it's now working?

**Comment by matanlurey** [comment]
Thanks @kailan, looks like it's WAI, appreciate it!
